.github/workflows/Documentation.yml

```diff
@@ -0,0 +1,32 @@
+# Adapted from
+# https://juliadocs.github.io/Documenter.jl/stable/man/hosting/#GitHub-Actions
+#
+# TODO: Do we need to install TagBot.yml for documenting tagged versions?
+# See discussion in the above URL regarding permissions for tagged release.
+
```

Comment:
It does not seem that we need TagBot.yml, so long as we maintain discipline in manually tagging each Github releas in our repo before adding it to the Julia Registry.